ManagerLib/UserControlEmulator.cs

```diff
@@ -0,0 +1,225 @@
+                if (!input)
+                {
+                    Console.ForegroundColor = ConsoleColor.DarkRed;
+                    Console.WriteLine("Unsupported command! Try again");
+                    Console.ResetColor();
+                    continue;
+                }
+                switch (MainMenuChoose)
+                {
+                    case 1:
+                        SeeAllSurveys();
+                        break;
+                    case 2:
+                        Console.ForegroundColor = ConsoleColor.DarkGreen;
+                        Console.WriteLine("See you next time!");
+                        Console.ResetColor();
+                        return;
+                    default:
+                        Console.ForegroundColor = ConsoleColor.DarkRed;
+                        Console.WriteLine("Unsupported command! Try again");
+                        Console.ResetColor();
+                        continue;
+                }
+            }
+        }
+
+        private void SeeAllSurveys()
+        {
+            foreach (var m in surveys)
+            {
+                Console.ForegroundColor = ConsoleColor.Cyan;
+                Console.WriteLine(m);
+                Console.ResetColor();
+            }
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("\nPlease enter id of survey, which you would like to take\n" +
+                "If you still change your mind about taking the survey, enter a negative id, for Example:\tID-> -1");
+            int parsed;
+            while (true)
+            {
+                Console.Write("ID-> ");
+                bool input = int.TryParse(Console.ReadLine(), out parsed);
+                if (!input)
+                {
+                    Console.ForegroundColor = ConsoleColor.DarkRed;
+                    Console.WriteLine("Wrong input param");
+                    Console.ResetColor();
+                    continue;
+                }
+                if (parsed < 0)
+                    return;
+                break;
+            }
+            bool surveyFound = false;
+            foreach (var m in surveys)
+            {
+                if (m.Id == parsed)
+                {
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    surveyFound = true;
+                    Console.WriteLine("\n" + m.ToString() + "\n");
+                    Console.ResetColor();
+                    for (var i = 0; i < m.QuestionContainer.Count; i++)
+                    {
+                        if (m.QuestionContainer[i].QuestionType == "OPEN")
+                        {
+                            Console.WriteLine($"{i + 1}.\t{m.QuestionContainer[i].Text}");
+                            while (true)
+                            {
+                                Console.Write("Ans-> ");
+                                var input = Console.ReadLine().TrimStart().TrimEnd();
```

Comment:
You can do it simplier: var input = Console.ReadLine().Trim();
(the same to var input = Console.ReadLine().TrimStart().TrimEnd();)